samples/client/others/typescript-axios/with-separate-models-and-api-inheritance/common.ts

```diff
@@ -85,6 +85,7 @@ export const setOAuthToObject = async function (object: any, name: string, scope
 }
 
 function setFlattenedQueryParams(urlSearchParams: URLSearchParams, parameter: any, key: string = ""): void {
+    if (parameter == null) return;
```

Comment:
This seems unrelated to what you changed?

Reply:
yeah, without this "samples up-to date" pipeline is failling